CoreScriptsRoot/CoreScripts/Topbar.lua

```diff
@@ -36,6 +36,10 @@ local function GetChatVisibleIconFlag()
 	local chatVisibleIconSuccess, chatVisibleIconFlagValue = pcall(function() return settings():GetFFlag("MobileToggleChatVisibleIcon") end)
 	return chatVisibleIconSuccess and chatVisibleIconFlagValue == true
 end
+
+local disableableTopbarSuccess, disableableTopbarFlagValue = pcall(function() return settings():GetFFlag("EnableSetCoreTopbarEnabled") end)
+local disableableTopbar = (disableableTopbarSuccess and disableableTopbarFlagValue == true)
```

Comment:
I'm not sure if disableable is a word, can we rename this variable so that it is a little more understandable?